pagopa/io-app
# feat(IT Wallet): [SIW-1036] Add eID detail screen

> [!WARNING]  
> This PR depends on #5780

## Short description

This PR adds the credential detail screen for the eID. The screen can support other credential types as well, it would just need a bit more fine tuning.

> [!Note]
> In the detail screen claims are grouped into sections, but this information is not included in the credential itself. To work around this, a mapping claim-section has been hardcoded in app. Claims that can not be assigned to any section are displayed after all the others.
> The **eID credential** used is also a **mock**.

## List of changes proposed in this pull request
- Extracted `RenderPidAssuranceLevel` and `RenderReleaserName` into separate components for easier reuse
- Added `ItwPresentationEidDetailScreen` component with related sub-components
- Modified `ItwCredentialClaim` to support hidden claims and dates with icon and badge
- Add utils in `itwClaimsUtils` to handle hardcoded claims configurations

## How to test
Navigate to **Profile > IT Wallet > Credential detail (eID)**. You should be able to see the eID detail screen.

## Preview

|iOS|Android|
|---|--------|
|<video src="https://github.com/pagopa/io-app/assets/52289599/03ab3b16-838f-4eb0-bbe4-006f1324692e">|<video src="https://github.com/pagopa/io-app/assets/52289599/7a7a7feb-f994-4669-9083-b398311d0086">|
